mapproxy/config/loader.py

```diff
@@ -1064,11 +1064,14 @@ def _file_cache(self, grid_conf, file_ext):
             log.warn('link_single_color_images not supported on windows')
             link_single_color_images = False
 
+        dimensionlist = self.context.globals.get_value('dimensions', self.conf,
```

Comment:
@olt recommended make `cache.dimensions` not a global option and this is reflected in RFC.